firebase-admin/runtime/src/main/java/io/quarkiverse/googlecloudservices/firebase/admin/runtime/FirebaseAuthConfig.java

```diff
@@ -47,6 +47,61 @@ public interface AuthConfig {
          */
         Optional<String> rolesClaim();
 
+        /**
+         * Returns the session cookie configuration
+         */
+        SessionCookie sessionCookie();
+
+        public interface SessionCookie {
+
+            /**
+             * Name to use for session cookies (see <a href="https://firebase.google.com/docs/auth/admin/manage-cookies#java_2">
+             * Manage session cookies</a>)
+             */
+            Optional<String> name();
+
+            /**
+             * The expiration duration of the session cookie. Uses {@link java.time.Duration#parse(CharSequence)} to
+             * get a duration for the expiration. See the JavaDoc of this method for the format of this value.
+             * <p>
+             * Defaults to 5 days.
+             */
+            @WithDefault("P5")
+            Optional<String> expirationDuration();
+
+            /**
+             * Perform an additional check to see if the session was revoked
+             */
+            @WithDefault("true")
+            Boolean checkRevoked();
+
+            /**
+             * Validate the expiration date of the token.
+             */
+            @WithDefault("false")
+            Optional<Boolean> validateToken();
+
+            /**
+             * Minimum token validity in case {@link #validateToken()} is set to true. Uses
+             * {@link java.time.Duration#parse(CharSequence)} to get a duration for the expiration. See the JavaDoc of
+             * this method for the format of this value.
+             */
+            Optional<String> minimumTokenValidity();
```

Comment:
You can directly use a Duration, there is a converter for it

Reply:
Ah cool, didn't know that, I'll update it.

Reply:
Fixed